src/app/services/character.service.ts

```diff
@@ -112,6 +112,21 @@ export class CharacterService {
     }
   }
 
+  /**
+   * Removes the item at the specified index from the inventory
+   */
+  removeItemFromInventory(index: number): void {
+    const inventory = this.$character.value.inventory;
+    inventory[index] = {
+      objectID: 0,
+      amount: 0,
+      variation: 0,
+      variationUpdateCount: 0
+    };
+    this.$character.next({ ...this.$character.value, inventory });
```

Comment:
Not needed. Only performance loss with this line

Reply:
Is actually needed. Inventory-Component doesn't update without it.